src/messages/manifestjson.js

```diff
@@ -7,3 +7,10 @@ export const MANIFEST_VERSION_INVALID = {
   description: _('See http://mzl.la/20PenXl (MDN Docs) for more information.'),
 };
 
+export const MANIFEST_NAME_INVALID = {
+  code: 'MANIFEST_NAME_INVALID',
+  legacyCode: null,
+  message: _('"name" in the manifest.json is not a valid value'),
+  description: _('See https://goo.gl/lSAJmB (MDN Docs) for more information.'),
```

Comment:
Ok it's working again today - I shall update. Not a great sign for a link shortener though.